composables/masto/icons.ts

```diff
@@ -11,17 +11,22 @@ export const accountFieldIcons: Record<string, string> = Object.fromEntries(Obje
   Facebook: 'i-ri:facebook-line',
   GitHub: 'i-ri:github-line',
   GitLab: 'i-ri:gitlab-line',
+  Gpg: 'i-ri:key-2-line',
```

Comment:
I think `GPG` (all uppercase) will be better

Reply:
I did think of that but they're all lowercased later anyway